Library/Homebrew/formula_installer.rb

```diff
@@ -368,11 +396,13 @@ def install_fetch_deps
     end
   end
 
+  sig { void }
   def build_bottle_preinstall
-    @etc_var_dirs ||= [HOMEBREW_PREFIX/"etc", HOMEBREW_PREFIX/"var"]
+    @etc_var_dirs = [HOMEBREW_PREFIX/"etc", HOMEBREW_PREFIX/"var"]
```

Comment:
i'm not sure i understand this an ivar, it appears to effectively be a constant

Reply:
It does seem to be. First added in b4f8671849e7bdac249c0d220cf25d054b956eaa, but that was to replace an ivar which was also effectively a constant (see a9764503b29a754762bfbfb4d4759a321cd29af1).

Reply:
Done! PTAL.